includes/modules/rstore-fl-product.php

```diff
@@ -84,6 +84,15 @@ public function __construct() {
 							'description' => __( 'Leave blank to hide button', 'reseller-store' ),
 							'default'     => __( 'Add to cart', 'reseller-store' ),
 						),
+						'button_new_tab'     => array(
+							'type'    => 'select',
+							'label'   => __( 'Open in new tab', 'reseller-store' ),
```

Comment:
**nitpick (non-blocking):** More inconsistency, casing is different to the other `Open in New Tab`